src/screens/SearchResults.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from "react";
+import map from "../assets/images/map.PNG";
+import { ListGroup } from "react-bootstrap";
+import CategoryButtonGroup from "../components/CategoryButtonGroup/CategoryButtonGroup";
+import EventLink from "../components/EventLink";
+import { clearAllBodyScrollLocks } from 'body-scroll-lock';
+import Map from "../components/maps/Map";
+import { withStore } from "../assets/helpers/store";
+import {withRouter, NavLink} from 'react-router-dom'
+
+class SearchResults extends Component {
+   constructor(props) {
+      super(props);
+      this.state = {
+         viewOnly: true,
```

Comment:
We have this `viewOnly` state in a lot of our components but we never use it for anything or set it to false. What is it used for?

Reply:
its used for the (unfinished) map stuff, it will be used to say that they can't open events from the mapview

Reply:
Cool